### PR TITLE
updated readme dependencies

### LINKED ISSUE #85: Dependencies need to be updated
In adaptivejs/assets/js/config.. If you add pinny you must also add these 3 packages in addition to those already listed in readme

 'event-polyfill': '../../../bower_components/pinny/src/js/utils/event-polyfil',
        'velocity': '../../../bower_components/mobify-velocity/velocity.min',
        'isChildOf': '../../../bower_components/selector-utils/src/selector/isChildOf',

@tedtate

### PR DESCRIPTION
**Pull request by dennislzf**
Fixed readme documentation where vital components of pinny were not referenced in require file
  
Issue: resolve #85 
 
Reviewers: @donnielrt @vmarta 

## Changes
- Added some dependencies to readme.md that need to be required in
  
## How to test-drive this PR
- add these new dependencies when adding pinny to project

### CODE REVIEW

**Comment by donnielrt** [comment]
Looping in @fractaltheory

**Comment by donnielrt** [comment]
Whoops, pressed the wrong button!

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ We highly recommend using Require.js with Pinny. To use Require, you have to ref
         'lockup': 'bower_components/lockup/dist/lockup.min',
         'deckard': 'bower_components/deckard/dist/deckard.min',
         'bouncefix': 'bower_components/bouncefix.js/dist/bouncefix.min'
+        'velocity': 'bower_components/mobify-velocity/velocity.min',
+        'isChildOf': 'bower_components/selector-utils/src/selector/isChildOf',
+        'sheet-top': 'bower_components/pinny/dist/effect/sheet-top.min'
```
Comment (fractaltheory) [inline README.md]: Looks like we've duplicated the `'sheet-top'` path here.

**Comment by fractaltheory** [comment]
When I try to use Pinny, I'm missing a reference to `event-polyfill` and it looks like the .js file is named `event-polyfil.js` so it can't find it. We should probably fix that.

**Comment by dennislzf** [comment]
We should probably fix that file name too sometime soon.

**Comment by fractaltheory** [comment]
Well, if you're already working on this PR ... :wink:

**Comment by Helen-Mobify** [comment]
Is this done? Can we merge this?

**Comment by dennislzf** [comment]
Yes I believe it is good to go

**Comment by fractaltheory** [comment]
@dennislzf could you ensure we have a package.json/bower.json version bump, CHANGELOG update, and github release pleases?

**Pull request merged**